mozilla/fx-private-relay
# Disable delete button when closing modal

This PR fixes #2194.

How to test:

1. Go to Relay page and log in using an existing account.
2. Go to the masks list and expand the card for one mask.
3. Click the red Delete button.
4. In the modal check the "Yes, I want to delete this mask." checkbox.
5. Click Cancel.
6. Click the red Delete button again.
7. Observe the Delete button in the modal.
